### DISCUSSION TITLE
operations' Batchwindow

### DISCUSSION THREAD

**Discussion by matteocarcano** [frePPLe/frepple#611] [Q&A] [answered]
Hi, 
i'm looking for the field "batchwindow" in the operations' table in the comunity edition but i'm not able to find it. I've found it in the trial enterprise edition. How can i manage this in the community edition? It's a foundamental feature.

Thank you

**Comment by hichamlahlou** [comment, marked as answer]
Hi,
The batch window field in operation table is a field used by the setup matrix feature.
The solver scans for the opportunity to create batches of MOs with the same setup within the specified batch window.
The setup matrix feature is only available on enterprise version, reason why you won't find this field in the community version of frePPLe.

**Comment by matteocarcano** [comment]
Maybe is there another parameter can let me to choice the time window until frepple aggregates requirements and makes only one manufacturing order? Otherwise i'm not undestanding how works the item's type field 'make to stock' : 

"make to stock (default):
Inventory, production and consumption of this item is managed aggregate"


Thank you for reply

**Comment by jdetaeye** [comment]
The make-to-stock / make-to-order flag is a very different topic. See https://frepple.com/docs/current/examples/buffer/make-to-order.php for examples on that.    This feature is about completely separating material flows that can't be mixed.

The batchwindow is about consolidating replenishments that are close to each other.  As Hicham already replied, that capability is only available in the Enterprise&Cloud Editions when using setup matrices.
